make.jl

```diff
@@ -0,0 +1,11 @@
+using Documenter
+
+makedocs(sitename = "JuliaAstro",
+    format = Documenter.HTML(prettyurls = get(ENV, "CI", nothing) == "true",
+        assets = [
+            "assets/favicon.ico",
+        ],
+    ),
+    strict = true
+)
+
```

Comment:
See the [`make.jl` script](https://github.com/JuliaPhysics/juliaphysics.github.io/blob/3e01bd7c43fbf43ca00414b784c06e44c126ee8d/make.jl) of `juliaphysics.github.io`, it has also the `deploydocs` function, to indicate which is the source branch (`source`), and which is branch where the pages will be saved to (`master`)